src/index.html

```diff
@@ -11,7 +11,137 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fielsed_field">
+        <legend>Personal information</legend>
+        <label class="label_field">
+          Surname
+          <input type="text"
+            name="surname"
+            required
+            minlength="4"
+            maxlength="16"
+            autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            name="surname"
            required
            minlength="4"
            maxlength="16"
            autocomplete="off"
          >
```
fix your code style according to the checklist

Reply:
fix all such cases